src/explore-education-statistics-admin/src/pages/release/data/components/ApiDataSetNotesForm.tsx

```diff
@@ -0,0 +1,51 @@
+import Button from '@common/components/Button';
+import ButtonGroup from '@common/components/ButtonGroup';
+import { Form, FormFieldTextArea } from '@common/components/form';
+import FormProvider from '@common/components/form/FormProvider';
+import LoadingSpinner from '@common/components/LoadingSpinner';
+import React from 'react';
+
+export interface ApiDataSetNotesFormValues {
+  notes: string;
+}
+
+export interface ApiDataSetCreateFormProps {
+  notes?: string;
+  onSubmit: (values: ApiDataSetNotesFormValues) => void;
+}
+
+export default function ApiDataSetNotesForm({
+  notes,
+  onSubmit,
+}: ApiDataSetCreateFormProps) {
+  return (
+    <FormProvider initialValues={{ notes }}>
+      {({ formState }) => {
+        return (
+          <Form id="apiDataSetCreateForm" onSubmit={onSubmit}>
+            <FormFieldTextArea
+              name="notes"
+              label="Public guidance notes"
+              hint="Use the public guidance notes to highlight any extra information to your end users that may not be apparent in the automated changelog below."
```

Comment:
This is a hella wordy. Could simplify this to: 'Highlight any extra information that may not be apparent in the automated changelog below.'